cpu/stm32f4/Makefile.include

```diff
@@ -1,5 +1,24 @@
 export CPU_ARCH = cortex-m4f
 export CPU_FAM  = stm32f4
 
+# STM32F4 uses sectos instead of pages, where the minimum length sector
+# is 16KB (the first sector), therefore RIOTBOOT must be 16KB to cover a whole sector
+RIOTBOOT_LEN ?= 0x4000
+
+# "The Vector table must be naturally aligned to a power of two whose alignment
+# value is greater than or equal to number of Exceptions supported x 4"
+# CPU_IRQ_NUMOFF for stm32l4 boards is < 102+16 so (118*4 bytes = 472 bytes ~= 0x200)
+# RIOTBOOT_HDR_LEN can be set to 0x200
+RIOTBOOT_HDR_LEN ?= 0x200
+
+# Sectors don't have the same length, they go from 16KB to 128KB. The first 5 sectors adding
```

Comment:
"The sum of the 5 first sectors is up to 128kB" might be more readable (if it's what you mean here).